testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/mail/basic/MailTesterServlet.java

```diff
@@ -66,8 +85,7 @@ protected void doGet(HttpServletRequest req, HttpServletResponse resp) throws Se
             if (!emailContent.equals(messages[0].getContent().toString().trim())) {
                 throw new ServletException("Message read is not equals to the message sent");
             }
-        } catch (MessagingException | InterruptedException e) {
-            throw new ServletException(e);
-        }
+
+        } catch (MessagingException | IOException ignored) {}
```

Comment:
@Dkafetzis you should not ignore them in the method, you have to ignore them outside, otherwise, as soon as any of these exceptions are ignored here, the test will pass.

Reply:
@yersan True, I've changed it